packages/marko-web-native-x/middleware/with-story.js

```diff
@@ -4,6 +4,8 @@ const createClient = require('../apollo/create-client');
 const defaultFragment = require('../apollo/graphql/fragments/story');
 const buildQuery = require('../apollo/graphql/queries/story');
 
+const gtmContainerId = process.env.NATIVE_X_GTM_CONTAINER_ID || 'GTM-PVHD8N5';
```

Comment:
Actually I changed my mind. Pass this in as a middleware option with the default value set.